helm/hpcc/values.schema.json

```diff
@@ -928,6 +928,16 @@
         "detail": {
           "type": "integer",
           "description": "Log output verbosity"
+        },
```

Comment:
Might be helpful to document these new options in the README, and PDF
[queueDropDefault ](https://github.com/hpcc-systems/HPCC-Platform/tree/master/helm/examples/logging#hpcc-systems-application-level-logging-details)

Reply:
Will add to the readme along with the others, but they're currently in helm/examples/logging/README.md.
examples/logging makes sense for setting up the logging stack setup, but the "logging:" settings would be better placed in helm/hpcc/README.md I think, as they are pertinent with or without a logging stack configuration.